modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -92,7 +92,11 @@ yarn remove @workday/canvas-kit-codemod
 
 ### Styling API and CSS Tokens
 
+<<<<<<< HEAD
+**PRs:** [#3101](https://github.com/Workday/canvas-kit/pull/3101), [#3088](https://github.com/Workday/canvas-kit/pull/3088), [#3114](https://github.com/Workday/canvas-kit/pull/3114), [#3119](https://github.com/Workday/canvas-kit/pull/3119)
+=======
 **PRs:** [#3101](https://github.com/Workday/canvas-kit/pull/3101), [#3088](https://github.com/Workday/canvas-kit/pull/3088), [#3114](https://github.com/Workday/canvas-kit/pull/3114), [#3120](https://github.com/Workday/canvas-kit/pull/3120)
```

Comment:
bad merge conflict, tis has happened to me

Reply:
Yeah I fixed that. Good to go, thanks for pointing it out!